content/en/docs/setup/production-environment/tools/kubespray.md

```diff
@@ -59,12 +59,12 @@ Kubespray provides the ability to customize many aspects of the deployment:
 * Choice deployment mode: kubeadm or non-kubeadm
 * CNI (networking) plugins
 * DNS configuration
-* Choice of control plane: native/binary or containerized with docker or rkt
+* Choice of control plane: native/binary or containerized with docker 
 * Component versions
 * Calico route reflectors
 * Component runtime options
   * {{< glossary_tooltip term_id="docker" >}}
-  * {{< glossary_tooltip term_id="rkt" >}}
+  * {{< glossary_tooltip term_id="containerd" >}}
   * {{< glossary_tooltip term_id="cri-o" >}}
 * Certificate generation methods (**Vault being discontinued**)
```

Comment:
I think we can remove the "Vault being discontinued", since it's not in the code base now.